tsconfig.json

```diff
@@ -26,7 +28,8 @@
 
     /* Strict Type-Checking Options */
     "strict": true /* Enable all strict type-checking options. */,
-    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
+    "noImplicitOverride": true /* Ensure overriding members in derived classes are marked with an explicit override modifier. */,
+    "noImplicitAny": true /* Raise error on expressions and declarations with an implied 'any' type. */,
```

Comment:
this is already enabled through `strict: true`

Reply:
oh, it is too!